.github/workflows/orquestra-sdk-integration.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow runs integration tests
```

Comment:
**nitpick**: could you paste here some of the reasoning you described in the PR description? The meaning of an _integration test_ is highly context-dependent.

For example, is the intention behind the tests here testing the integration between our packages (I guess this is the use case here), or the integration between components used _inside_ a single package (we have a `test_integration.py` file that starts Ray runtime and executes real workflows).